ecms_base/ecms_base.install

```diff
@@ -44,4 +44,21 @@ function standard_install() {
   ]);
 
   $shortcut->save();
+
+  // Assign the base content types to our workflow.
+  /** @var \Drupal\workflows\WorkflowInterface $workflow */
+  $workflow = \Drupal::entityTypeManager()
```

Comment:
Knowing that every content type will need to be added to the editorial workflow. Would it make sense to offload this functionality to a custom service in a module that implements [hook_entity_bundle_create()](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21entity.api.php/function/hook_entity_bundle_create/8.2.x)? Then, no matter what content type gets created in the future, they will all get added to the editorial workflow.

Reply:
That sounds like it could work, however my understanding of the scope was that a few of the content types would not fall under this workflow, such as notifications and landing pages. We will need a solution for the non-base content types  since those will be enabled separately, so I can look into your proposed solution to handle those regardless.
For the short term, I was unable to get the moderation dashboard running without adding at least one content type to the editorial workflow, so this allowed the branch to move forward.

Reply:
@pfrilling Ok - introduced a new custom module to handle this. It will apply to all content types except for notifications.